### PR TITLE
Add minimum to body in proposal schema

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/schemas/proposal.json [hunk 1]
```diff
@@ -56,10 +56,7 @@
           "type": "number",
           "title": "end",
           "minimum": 1000000000,
-          "maximum": 2000000000,
-          "exclusiveMinimum": {
-            "$data": "1/start"
-          }
```
Comment (ChaituVR) [inline src/schemas/proposal.json:62]: Oh, this never works 🥲 Is there some other way to limit this?

**Review by ChaituVR — APPROVED** [review]
tAck

File: src/schemas/proposal.json (hunk 1 above)
Comment (ChaituVR) [reply src/schemas/proposal.json:62]: will handle on seq

Comment (samuveth) [reply src/schemas/proposal.json:62]: Thanks :) Yeah this was causing error with newer versions.

Comment (samuveth) [reply src/schemas/proposal.json:62]: And I couldn't find a solution too do it in another way on the schema. Sorry should have mentioned it in the proposal description

**Pull request merged**